packages/web/src/features/languages/ImportLanguageGlossesView.tsx

```diff
@@ -119,8 +122,8 @@ export default function ImportLanguageGlossesView() {
 
   return (
     <>
-      <View fitToScreen className="flex justify-center items-start">
-        <div className="mx-4 flex-shrink">
+      <View fitToScreen className="flex items-start justify-center">
+        <div className="flex-shrink mx-4">
```

Comment:
Ah yes, I did. Sorry about that. I've removed it now.

Reply:
There is a prettier tailwind plugin for ordering classes, it would be great to add.

https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

Reply:
@Pertempto Would that be another issue?